content/events/22-23/lan.md

```diff
@@ -6,14 +6,34 @@ end: '22-10-2022 03:00'
 location: Therminal, Hoveniersberg 24, 9000 Gent
 description: Zeus' elfde LAN-Party!
 header_text_background: true
-banner: 
+banner: https://pics.zeus.gent/zTVVLv1sQAlbS1WFZG60zQQbk4eOjvWIPudB9Uhd.jpg
 locationlink: $therminal
 ---
 
-# Geeks Attack XI
+# Do you wanna play a game ?
+Het is weer de tijd voor de jaarlijkste LAN party van Zeus WPI. Voor iedereen die tot in de laatste uurtjes gezamelijk wilt gamen.
 
-Op **21 oktober** organiseert Zeus zijn elfde LAN-Party: **Geeks Attack XI**!
+## Info
 
-Details worden later nog aangevuld, aanvangstijd kan wijzigen.
+**Datum**: Vrijdag 21 oktober - Zaterdag 22 oktober
+
+**Tijd**: 18:00 tot wanneer we niet meer kunnen
+
+**Locatie**: Therminal, Hoveniersberg 24, Gent
+
+**Meenemen**: Stekkerdoos + ethernetkabel
+
+**Inschrijven**: Je moet je niet inschrijven voor de lan zelf. Info volgt i.v.m. competitions
+
+## Competitions
+
+Trackmania, Minecraft, Mario Kart, Nidhogg en nog veel meer!
+
+## Prijzen
+
+Er zijn in totaal honderden euro's aan prijzen, dus je hoeft niet eens de beste te zijn om met iets naar huis te gaan!
 
 Tot dan!
+Zeus WPI
+
+![Lan party poster](https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg "Lan party poster")
```

Comment:
```suggestion
![Lan party poster](https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg "Lan party poster"){ width=60% }
```

assuming dat we pandoc gebruiken (https://stackoverflow.com/a/41169162/19735148), mss dat dit iets overzichtelijker gaat zijn.

Reply:
Ik denk dat dat nie gaat werken, volgens mij moet je met een `<img/>` werken op github

Reply:
welja, maar de site wordt niet door GitHub gehost right? 't is enkel de preview hier dat niet zou werken dan bv. Maar dit was puur een poging, dus geen idee of dit op de site gaat werken

Reply:
Kzal eens dit proberen:
`<img src="https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg" max-width=700px />`

Reply:
Aaahja je hebt gelijk haha toch jouw ding proberen

Reply:
> Kzal eens dit proberen:
> `<img src="https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg" max-width=700px />`

Moest je laatste commit niet werken, dan zou dit werken, alleen moet je dat in een `style` zetten, dus 
`<img src="https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg" style="max-width: 700px;" />`